PREVIEW-FEATURES.md

```diff
@@ -0,0 +1,45 @@
+# Preview features
```

Comment:
is this a good location of this document?
should it be in /doc-files somewhere?

Reply:
I think it's not a documentation of JavaFX itself, but more like a meta-documentation like our contribution guidelines.